cli/upload.py

```diff
@@ -123,46 +123,63 @@ def _open_file_mapping(extra_metadata: dict, base_path: str) -> Dict[str, Binary
 )
 
 
-def _start_procs(upload_info: api.UploadInfo, xlsx: str) -> list:
+def _start_procs(srs_dst_pairs: list) -> list:
```

Comment:
`src_dst_pairs`? Also, a note in the docstring that this is a list of tuples containing local file paths and target GCS paths would be useful.